src/Editor/GUI/Editors/EntityEditor.cpp

```diff
@@ -585,3 +616,40 @@ void EntityEditor::VRDeviceEditor(Component::VRDevice* vrDevice) {
         ImGui::Unindent();
     }
 }
+
+
+void EntityEditor::SetVertexData(Geometry::AssetFileHandler::MeshData * data) {
+    
+    vertices = new Video::Geometry::VertexType::StaticVertex[data->numVertices];
+    indices = new uint32_t[data->numIndices];
+    for (int i = 0; i < data->numVertices; i++) {
+        vertices[i] = data->staticVertices[i];
+        nrOfVerts = data->numVertices;
+        nrOfIndices = data->numIndices;
+    }
+    for (int i = 0; i < data->numIndices; i++) {
+        indices[i] = data->indices[i];
+    }
+
+}
+
+int EntityEditor::GetNrOfVerts() {
+    return nrOfVerts;
+}
+
+int EntityEditor::GetNrOfIndices() {
+    return nrOfIndices;
+}
+
+Video::Geometry::VertexType::StaticVertex* EntityEditor::GetVertices() {
+    return vertices;
+}
+
+bool EntityEditor::isLoaded() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'isLoaded' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)